bin.js

```diff
@@ -98,8 +98,9 @@ program
 	.option('-m, --max <value>', 'Maximum number of results', 10)
 	.option('-b, --minBlock <value>', 'The smallest block to include, if any')
 	.option('-s, --synth <value>', 'Synth code')
-	.action(async ({ max, synth, minBlock }) => {
-		rate.updates({ max, synth, minBlock }).then(console.log);
+	.option('-t, --minTimestamp <value>', 'The oldest timestamp to include')
```

Comment:
ah would you mind making these `min` and `max` and supporting both?

Reply:
should we add `maxBlock` then?